draft-ietf-webtrans-http2.md

```diff
@@ -1519,4 +1563,4 @@ Comments:
 
 Thanks to Anthony Chivetta, Eric Gorbaty, Ankshit Jain, Joshua Otto, and
 Valentin Pistol for their contributions in the design and implementation of
-this work.
+this work. The requirements on TLS usage were inspired by {{?RFC9729}}.
```

Comment:
I'm not sure that you need to say that.  You could instead acknowledge yourself.

Reply:
Well that would just be uncouth